manifests/prometheus-prometheus.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: monitoring.coreos.com/v1
 kind: Prometheus
 metadata:
   labels:
+    app.kubernetes.io/component: database
```

Comment:
Hah I would say Prometheus is a bit more than a database, e.g. it also scrapes and exposes an UI? :)

Reply:
I had hard time assigning promtheus into a one-word bucket. For me, it should be either `prometheus` or `database` and since from administration pov it behaves like a database I went with that.

Reply:
Yeah, no clue what is usually put there, and who would find this info useful?

Reply:
Usually, this reflects what is the component role in the stack. AFAIK this one isn't very important/useful and it is mostly for logical grouping.

Reply:
according to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels `app.kubernetes.io/component` should reflect the "The component within the architecture".

As suggested above prometheus includes quite many component responsibility within our architecture, one being `tsdb`, another being `frontend`, yet another one being `alerting`.

How about `monitoring-system`? (I am out of ideas as well)

Reply:
Yeah I would say monitoring-system as well, its at least more accurate and not misleading? But really out of ideas as well. @ant31 what did you envision for this?

Reply:
The goal is to give info to user about what this resource is doing in the context of the 'part-of' label. 
For example: front-end/backend  etc, so even for people who don't know the application they would have a clue of what this component does and how it fit in the system. 
 
I guess 'tsdb' would be good,
Or since prometheus is famous, 'prometheus' could work also. 

There isn't any recommended set of values, you decide :)

Reply:
I don't agree with `monitoring-system`, the whole thing we deploy here is a monitoring system where prometheus is just one of the components. I changed this to `prometheus`.